### ISSUE TITLE
`stopb()` will fail on Linux systems due to use of `taskkill` command

### ISSUE THREAD

**Issue by stvnliu** [Ate329/MinecraftBot123#2]
As title suggests, `node.exe` and `taskkill` are both not found in a Linux system. This will cause the function to fail when invoked. :(
[Failing line](https://github.com/Ate329/MinecraftBot123/blob/0d1b8cbc5bb28d1dd09b4c07935d374cc9a2e3ce/bot.py#L155)
```python
def stopb():
    os.system('taskkill /f /im node.exe')
    strtb.join()
```

**Comment by stvnliu** [comment]
@Ate329 Replacing all `threading.Thread` with `multiprocessing.Process` allows a proper `SIGTERM` to be sent to the process, thus shutting down gracefully.

**Comment by Ate329** [comment]
Solved in PR #5
